### PR TITLE
fix: update timestamp display to use uniform number of chars

### PR DESCRIPTION
**Pull request by aaron-steinfeld**

### CODE REVIEW

File: projects/common/src/utilities/formatters/date/date-formatter.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ export class DateFormatter {
         return 'd MMM y h:mm a';
       case DateFormatMode.DateAndTimeWithSeconds:
       default:
-        return 'y-M-d hh:mm:ss a';
+        return 'y-MM-dd hh:mm:ss a';
```
Comment (jake-bassett) [inline projects/common/src/utilities/formatters/date/date-formatter.ts:57]: Oh thank god.

Is the cell renderer using this so it will be fixed as well?

Comment (aaron-steinfeld) [reply projects/common/src/utilities/formatters/date/date-formatter.ts:57]: sure is

Comment (aaron-steinfeld) [reply projects/common/src/utilities/formatters/date/date-formatter.ts:57]: sure is.

Comment (anandtiwary) [reply projects/common/src/utilities/formatters/date/date-formatter.ts:57]: sure is..

**Pull request merged**